src/task/MonitorTask.ts

```diff
@@ -235,6 +228,23 @@ export default class MonitorTask
       return;
     }
 
+    const metricReport = clientMetricReport.getObservableMetrics();
+    /* istanbul ignore else */
+    if (this.context.videoTileController.hasStartedLocalVideoTile()) {
+      let videoUpstreamBitrateKbps = 0;
+      const videoUpstreamPacketPerSecond = metricReport.videoPacketSentPerSecond;
+      const videoUpstreamBitrate = metricReport.videoUpstreamBitrate;
+
+      /* istanbul ignore else */
+      if (typeof videoUpstreamBitrate === 'number' && !isNaN(videoUpstreamBitrate)) {
+        videoUpstreamBitrateKbps = videoUpstreamBitrate / 1000;
+      }
+
+      this.videoSendHealthDidChange(videoUpstreamBitrateKbps, videoUpstreamPacketPerSecond);
```

Comment:
Since, we are removing this from AudioVideoObserver, should we make this a private method in MonitorTask?

Reply:
Yes, I didn't realize it wasn't

Reply:
Can we move this entire logic to its own function with a more description name?

Reply:
Updated and changed name to be `updateSendingVideoStats`

Reply:
We decided to remove this function